unit_tests/test_tester.py

```diff
@@ -109,89 +95,35 @@ def start_argus_heartbeat_thread(self):
-                f"({total_events}) expected:\n{''.join(events_by_category[event_category])}"
-            assert final_event.events[event_category][0] == events_by_category[event_category][-1]
-        if final_event.test_status == 'SUCCESS':
-            assert unittest_final_event is None
-            assert str(final_event) in sct_log
-        else:
-            assert isinstance(unittest_final_event, TestResultEvent)
-            assert str(final_event) not in sct_log
 
     @property
     def final_event(self) -> TestResultEvent:
-        if self._final_event:
-            return self._final_event
-        self._final_event = self.unittest_final_event
-        if self._final_event:
-            return self._final_event
-        self._final_event = self._get_test_result_event()
-        return self._final_event
-
-    @property
-    def unittest_final_event(self) -> TestResultEvent:
-        if self._unittest_final_event is not False:
-            return self._unittest_final_event
-        final_event = self._get_unittest_final_event()
-        self._unittest_final_event = final_event
-        return final_event
-
-    def _get_unittest_final_event(self) -> TestResultEvent:
-        errors = self._outcome.errors.copy()
-        for error in errors:
-            if error and error[1] and error[1][1] and isinstance(error[1][1], TestResultEvent):
-                return error[1][1]
-        return None
-
-    @property
-    def sct_log(self):
-        if self._sct_log:
-            return self._sct_log
-        with open(os.path.join(self.logdir, 'sct.log'), encoding="utf-8") as log_file:
-            output = log_file.read()
-        self._sct_log = output
-        return output
+        return self._get_test_result_event()
 
     @property
     def event_summary(self) -> dict:
-        if self._event_summary:
-            return self._event_summary
-        self._event_summary = self.get_event_summary()
-        return self._event_summary
+        return self.get_event_summary()
 
     @property
     def events(self) -> dict:
-        if self._events:
-            return self._events
-        self._events = get_events_grouped_by_category(_registry=self.events_processes_registry)
-        return self._events
-
-    def stop_event_device(self):
-        time.sleep(0.5)
-        super().stop_event_device()
+        return get_events_grouped_by_category(_registry=self.events_processes_registry)
 
     def save_schema(self):
         pass
 
+    @pytest.fixture(autouse=True, name='event_system')
+    def fixture_event_system(self, setup_logging):
+        # Cover multi-tenant configuration. Prevent event device double initiate
+        start_events_device(log_dir=self.logdir,
+                            _registry=self.events_processes_registry)
+        yield
+        self.stop_event_device()
+        events_processes._EVENTS_PROCESSES = None
+
 
 class SubtestAndTeardownFailsTest(ClusterTesterForTests):
+    __test__ = True
```

Comment:
How about renaming the class instead? Or changing the naming scheme in pytest config?

Reply:
it's not related to pytest, we disable on `ClusterTesterForTests`, we need to enable it, for this to work